yt_dlp/extractor/ixigua.py

```diff
@@ -0,0 +1,118 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+    int_or_none,
+    js_to_json,
+    str_or_none,
+    traverse_obj,
+)
+import base64
+
+
+class IxiguaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:\w+\.)?ixigua\.com/(?:video/)?(?P<id>[0-9]+).+'
+    _TEST = {
+        'url': 'https://www.ixigua.com/6996881461559165471',
+        'info_dict': {
+            'id': '6996881461559165471',
+            'ext': 'mp4',
+            'title': '盲目涉水风险大，亲身示范高水位行车注意事项',
+            'description': '本期《懂车帝评测》，我们将尝试验证一个夏日大家可能会遇到的关键性问题：如果突发暴雨，我们不得不涉水行车，如何做才能更好保障生命安全。',
+            'tag': 'video_car',
+            'like_count': int,
+            'dislike_count': int,
+            'view_count': int,
+            'uploader': '懂车帝原创',
+            'uploader_id': '6480145787',
+            'thumbnail': r're:^https?://.*\.(avif|webp)(?:\?.+)',  # still not sure for regex
+            'timestamp': 1629088414,
+            'duration': 1030,
+        },
+        'skip': 'This Extractor need cookies',
```

Comment:
```suggestion
```

Reply:
Is it fine to do this ? The extractor need cookies to work, the test will be expected to fail